src/main/java/uz/bookstore/bookstore/controller/AddressController.java

```diff
@@ -0,0 +1,59 @@
+package uz.bookstore.bookstore.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uz.bookstore.bookstore.dto.AddressInDTO;
+import uz.bookstore.bookstore.dto.AddressOutDTO;
+import uz.bookstore.bookstore.service.AddressService;
+
+import java.util.List;
+
+@RequestMapping("/api/v1/address")
+@RestController
+@RequiredArgsConstructor
+public class AddressController {
+    private final AddressService addressService;
+
+    @PostMapping
+    public ResponseEntity<?> addAddress(@RequestBody AddressInDTO addressDTO) {
+        AddressOutDTO result = addressService.addAddress(addressDTO);
```

Comment:
We do not need to write **AddressOutDTO result = addressService.addAddress(addressDTO);**
we should write return **ResponseEntity.ok(addressService.addAddress(addressDTO);**
because we do not user **result** anymore.
Please fix all methods.